infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -817,7 +818,9 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
         address to,
         uint256 amount
```

Comment:
Should we pass in the cost here so the average cost variable in the contract will remain accurate?